src/invidious/config.cr

```diff
@@ -319,6 +319,18 @@ class Config
       end
     end
 
+    # Check if the domain configuration is valid
+    if (domain = config.domain).nil? || domain.empty?
+      puts "Config: 'domain' is required/can't be empty"
+      exit(1)
+    elsif parsed_domain = URI.parse(domain)
+      if !parsed_domain.scheme.nil?
+        puts "Config: 'domain' cannot contain a scheme (#{parsed_domain.scheme})"
+        puts "Config: (Hint) Set 'domain' to '#{parsed_domain.host}'"
```

Comment:
Even though the scheme is an usual mistake, we should probably catch all other kinds of errors (trailing `/`, port number, etc)...
```suggestion
      if domain != parsed_domain.domain
        puts "Config: 'domain' is invalid.
        
        if parsed_domain.host.presence
          puts "Config: (Hint) Did you mean #{parsed_domain.host} ?"
        else
          puts "Config: (Hint) 'domain' should look like this: invidious.example.com"
        end
```